conf/config.go

```diff
@@ -31,6 +31,7 @@ type Configuration struct {
 	METADATA_DB_HOST        string
 	METADATA_DB_PORT        string
 	METADATA_DB_TLS_ENABLED bool
+	METADATA_DB_TLS_MUTUAL  bool
```

Comment:
This note is for @valeraBr.
When this PR will be merged, please add the option to set it from the Helm values and pass it as an environment variable.